comps/comps-foreman-rhel7.xml

```diff
@@ -327,11 +329,7 @@
         Do not edit this section manually and use ./comps_doc.sh script
       -->
       <!--RGD-START-->
-      <packagereq type="default">nodejs-expose-loader-doc</packagereq>
-      <packagereq type="default">nodejs-file-loader-doc</packagereq>
       <packagereq type="default">nodejs-jquery-doc</packagereq>
-      <packagereq type="default">nodejs-jquery-flot-doc</packagereq>
-      <packagereq type="default">nodejs-select2-doc</packagereq>
```

Comment:
This was done by `comps_doc.sh`, should I put those back?

Reply:
ummmm, my guess is no, @ekohl?

Reply:
We dropped various packages but probably didn't regenerate the comps file. Probably cleanest to fix this in `rpm/develop` and rebase this PR on it.

Reply:
Opened https://github.com/theforeman/foreman-packaging/pull/4195